Stylesheet/Components/grid.css

```diff
@@ -0,0 +1,15 @@
+/* This is grid file */
+
+.grid2 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 0.5rem;
```

Comment:
You can keep the **display:grid** and **grid-gap:0.5rem** in another class for better modularity

Reply:
Sure changing that, thank you